jobs/scripts/glusto/setup-glusto.yml

```diff
@@ -96,6 +96,15 @@
       gpgcheck: yes
       gpgkey: https://copr-be.cloud.fedoraproject.org/results/nigelbabu/arequal/pubkey.gpg
       repo_gpgcheck: no
+  
+  - name: Setup the repo
+    yum_repository:
+      name: gluster_nightly
+      description: The nightly repo for Gluster
+      #baseurl: "{{ 'http://' + hostvars[groups['gluster_nodes'][0]]['ansible_eth0']['ipv4']['address'] + '/RPMS/' }}"
```

Comment:
I think we can remove it.

Reply:
> I think we can remove it.

Yes, removed the line with commented #baseurl